bukkit/src/main/java/de/maxhenkel/voicechat/config/ServerConfig.java

```diff
@@ -30,79 +30,79 @@ public ServerConfig(ConfigBuilder builder) {
+                        "The maximum size that voice packets are allowed to have (in bytes)"
                 );
         keepAlive = builder
                 .integerEntry("keep_alive", 1000, 1000, Integer.MAX_VALUE,
-                        "The frequency in which keep alive packets are sent",
+                        "The frequency at which keep-alive packets are sent (in milliseconds)",
                         "Setting this to a higher value may result in timeouts"
                 );
         groupsEnabled = builder
                 .booleanEntry("enable_groups", true,
-                        "If group chats are allowed"
+                        "Controls whether group chats are allowed"
                 );
         voiceHost = builder
                 .stringEntry("voice_host", "",
-                        "The host name that clients should use to connect to the voice chat",
+                        "The hostname that clients should use to connect to the voice chat",
                         "This may also include a port, e.g. 'example.com:24454'",
-                        "Don't change this value if you don't know what you are doing"
+                        "Do NOT change this value if you don't know what you're doing"
                 );
         allowRecording = builder
                 .booleanEntry("allow_recording", true,
-                        "If players are allowed to record the voice chat"
+                        "Controls whether players are allowed to record audio"
                 );
         spectatorInteraction = builder
                 .booleanEntry("spectator_interaction", false,
-                        "If spectators are allowed to talk to other players"
+                        "Controls whether spectators are allowed to talk to other players"
                 );
         spectatorPlayerPossession = builder
                 .booleanEntry("spectator_player_possession", false,
-                        "If spectators can talk to players they are spectating"
+                        "Controls whether spectators can talk to players they are spectating"
                 );
         forceVoiceChat = builder
                 .booleanEntry("force_voice_chat", false,
-                        "If players without the mod should get kicked from the server"
+                        "Controls whether players without this mod should be kicked from the server"
                 );
         loginTimeout = builder
                 .integerEntry("login_timeout", 10_000, 100, Integer.MAX_VALUE,
-                        "The amount of milliseconds, the server should wait to check if the player has the mod installed",
-                        "Only active when force_voice_chat is set to true"
+                        "The amount of time the server should wait to check if a player has the mod installed (in milliseconds)",
+                        "Only relevant when 'force_voice_chat' is set to 'true'"
```

Comment:
Please use double quotes for consistency

Reply:
This is no longer relevant, please refer to a different conversation thread instead.